Decisions.Box/BoxSettings.cs

```diff
@@ -342,6 +342,10 @@ public ValidationIssue[] GetValidationIssues()
         {
             if (JsonConfig == null)
                 issues.Add(new ValidationIssue(JsonConfig, "You must provide the JSON configuration file provided in your administration console."));
+            
+            string extension = System.IO.Path.GetExtension(JsonConfig?.FileName) ?? "";
+            if (!extension.Contains("json", StringComparison.OrdinalIgnoreCase))
+                issues.Add(new ValidationIssue(this, "Unexpected file extension; file extension is not JSON"));
```

Comment:
"Unexpected File Extension. The Configuration File must be a valid JSON file with a .json extension."

Reply:
Different message.... gotcha.  I'll update